txp/Pos/Txp/Core/Types.hs

```diff
@@ -130,7 +130,7 @@ type TxWitness = Vector TxInWitness
 -- be the same as `txOutValue` of corresponding output.
 type TxOutDistribution = [(StakeholderId, Coin)]
 
--- | Distribution of “fake” stake that follow-the-satoshi would use
+-- | Distribution of "fake" stake that follow-the-satoshi would use
```

Comment:
So what about this? Personally I would remove the word `fake` altogether, because a lot of things changed and I wouldn't call it `fake` now.